src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -3922,6 +3922,37 @@ public static String join(final boolean[] array, final char delimiter) {
      * @since 3.12.0
      */
     public static String join(final boolean[] array, final char delimiter, final int startIndex, final int endIndex) {
+        return join(array, String.valueOf(delimiter), startIndex, endIndex);
+    }
+
+    /**
+     * <p>Joins the elements of the provided array into a single String containing the provided list of elements.</p>
```

Comment:
The first sentence in a Javadoc comment is special, it is the summary and should not be additionally wrapped in a paragraph.

Reply:
Thanks for pointing this out, I didn't know about such rule. 
Additional paragraph tags removed.